pkg/liqonet/ipam/ipam.go

```diff
@@ -989,10 +989,13 @@ func (liqoIPAM *IPAM) mapEndpointIPInternal(clusterID, ip string) (string, error
 		return "", fmt.Errorf("cannot get cluster subnets:%w", err)
 	}
 	subnets, exists = clusterSubnets[clusterID]
+	if !exists {
+		return "", fmt.Errorf("cluster %s has not a network configuration", clusterID)
+	}
 
 	// Get PodCIDR
 	podCIDR, err := liqoIPAM.ipamStorage.getPodCIDR()
-	if err != nil {
+	if err != nil || podCIDR == "" {
```

Comment:
Would be nice to have "" mapped to a constant which explains better the semantic of the value. Something like "EmptyPODCIDR" or similar stuff.